core/conversion/converters/Weights.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "core/util/prelude.h"
+#include "core/conversion/conversionctx/ConversionCtx.h"
+
+namespace trtorch {
+namespace core {    
+namespace conversion {
+namespace converters {
+
+struct Weights {
+    //TODO: Rebuild this in a way that makes sense for more than just conv2/3D and linear
+    nvinfer1::Weights data;
+    nvinfer1::Dims kernel_shape;
+    nvinfer1::Dims shape;
+    int64_t num_input_maps;
+    int64_t num_output_maps;
+
+    Weights();
+    Weights(ConversionCtx* ctx, at::Tensor t);
+    Weights(ConversionCtx* ctx, float val);
+    Weights(ConversionCtx* ctx, int32_t val);

```

Comment:
Do we need on for bool too?

Reply:
I removed it because a IConstantLayer doesn't accept a bool Weight/Tensor anyways. The correct-use case for creating a bool Tensor is to pass the IConstantLayer through an identity layer, and manually set the output to be of DataType::kBOOL. 

Let me know what you think.

Reply:
I'm going to keep it out for now. We can add one in if we need to have a Weight (but not a Tensor) with a single bool.

Reply:
ok